lib/rules/max-statements-per-line.js

```diff
@@ -35,11 +35,12 @@ module.exports = {
         const sourceCode = context.getSourceCode(),
             options = context.options[0] || {},
             maxStatementsPerLine = typeof options.max !== "undefined" ? options.max : 1,
-            message = "This line has too many statements. Maximum allowed is {{maxStatementsPerLine}}.",
-            data = { maxStatementsPerLine };
+            emptyBlockMessage = "This line has an empty block, but no statements are allowed.",
```

Comment:
That would mean it says "there are 0 statements, the max is 0" - which makes no sense either. :-/ I think something special has to be done here.